seatunnel-connectors-v2/README.md

```diff
@@ -2,10 +2,11 @@
 This article introduces the new interface and the new code structure on account of the newly designed API for Connectors in Apache SeaTunnel. This helps developers with quick overview regarding API, translation layer improvement, and development of new Connector.
```

Comment:
SeaTunnel has designed a new connector API to decouple from the compute engine. This article introduces the new interface and the new code structure to help developers quickly get started with the new API and understand the principles of the new API.